src/linear_algebra/affine_space/midpoint.lean

```diff
@@ -107,6 +107,12 @@ right_vsub_midpoint v₁ v₂
 
 variable (R)
 
+@[simp] lemma midpoint_eq_left_iff {x y : P} : midpoint R x y = x ↔ x = y :=
+by rw [midpoint_eq_iff, point_reflection_self]
+
+@[simp] lemma midpoint_eq_right_iff {x y : P} : midpoint R x y = y ↔ x = y :=
+by rw [midpoint_comm, midpoint_eq_left_iff, eq_comm]
```

Comment:
Should we also have lemmas about `x = midpoint R x y` and `y = midpoint R x y`? Not sure. Otherwise LGTM. Thanks!
bors d+